test/streams/streams_controller_test.rb

```diff
@@ -23,6 +23,7 @@ class Turbo::StreamsControllerTest < ActionDispatch::IntegrationTest
       <turbo-stream action="replace" target="message_1"><template>#{render(message_1)}</template></turbo-stream>
       <turbo-stream action="replace" target="message_1"><template>Something else</template></turbo-stream>
       <turbo-stream action="replace" target="message_5"><template>Something fifth</template></turbo-stream>
+      <turbo-stream method="morph" action="replace" target="message_5"><template>Something fifth</template></turbo-stream>
```

Comment:
Maybe also cover the `action="update"` case in exactly the same way?

Reply:
update not covered at all as you see but anyway added with morph :)